Gemfile

```diff
@@ -69,6 +69,7 @@ end
 group :test do
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223336687-eab5b8ca-4a46-4bcf-a158-cb5ca1d56f43.png)

- [ ] Kindly remove those links which are not working to make your readme file descriptive and clean :blush: